src/utils/SortUtil.ts

```diff
@@ -0,0 +1,14 @@
+export enum SortOrder {
+  ASC = 'asc',
```

Comment:
not too important,

but introducing the concept of sorting is fine to me, but why not then add to the other places as well? credentials? identities?

now only contacts are sorted, which does not make much sense to me if you spent time on getting sorting implemented

can we atleast put some comments on the other places and create a ticket for it?

Reply:
Other places are not part of this ticket. I would request to not expand. and let this ticket go towards closure. 

I will create a ticket just as a reminder.
https://sphereon.atlassian.net/browse/WAL-605

Reply:
The sorting for contacts was also not part of the ticket. lets add some todo's on the other places that references the ticket you made

Reply:
I was trying to avoid even the references in the code. But I will shortly do add those.

There were 2 rules.
1- Don't change anything that is not part of the ticket.
2- If the ticket inroduces a bug it will be fixed in the same ticket. Since, the changes in the Contact reducer were made and it was seen that the over all combination of running different reducers resulted in different ordering. Hence, this change was made. 

I opted the 2nd rule.

Also, in previous ticket(s) the "other" developer also makes such small changes. As the code-author wills.

Reply:
Yeah, but either do it right then or don't do it

Reply:
So do it all, or create a ticket. Not do partially and be done with it

Reply:
Then there will be comments about intorudcing the sorting issue and not fixing it. 
So there are 3 conflicting rules.
Winner is the most persistent disagreeable one.